src/components/DeploymentList.vue

```diff
@@ -75,14 +75,17 @@
       </template>
 
       <template #action="{ row }">
-        <DeploymentMenu
-          class="deployment-list__menu"
-          small
-          show-all
-          :deployment="row"
-          flat
-          @delete="refresh"
-        />
+        <div class="deployment-list__action">
```

Comment:
Not blocking, in case you weren't aware - if you don't want to wrap this with a div + new class, you can use the existing column class passed at the table root `.deployment-list__action-column`